app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,23 @@
+class RestaurantsController < ApplicationController
+
+  def index
+    render json: Restaurant.all
+  end
+
+  def show
+    render json: Restaurant.find(params.require(:id))
+  end
+
+  def create
+    restaurant = Restaurant.create!(name: params.require(:name),
+                          address: params.require(:address),
+                          does_accept_10bis: params.require(:does_accept_10bis),
+                          delivery_sla_in_minutes: params.require(:delivery_sla_in_minutes),
+                          cuisine_id: params.require(:cuisine_id))
```

Comment:
I personally suggest sending all the parameters under one object, for example:
```html
<input name="restaurant[foo]" ... />
```

And have a method along these lines:
```ruby
def restaurant_params
  params.require(:restaurant).permit(:name, :address, ...)
end
```

This is a pattern suggested by the documentation: http://guides.rubyonrails.org/action_controller_overview.html#strong-parameters

Reply:
even if you're not doing this, you can just use: `Restaurant.create!(params.require(:name, : address...)`